api/v1alpha1/proxmoxmachine_types.go

```diff
@@ -106,7 +106,13 @@ type ProxmoxMachineSpec struct {
 	// +kubebuilder:validation:XValidation:rule="self.end >= self.start",message="end should be greater than or equal to start"
 	VMIDRange *VMIDRange `json:"vmIDRange,omitempty"`
 
+	// Checks defines possibibles checks to skip.
```

Comment:
typo

Reply:
Suggested by Copilot, you can submit an issue 😄

Reply:
you need to regen.